src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -47,13 +53,23 @@
 /** Tests the User Data Servlet */
 @RunWith(JUnit4.class)
 public final class UserDataServletTest {
+  private static final String PROFILE_PHOTO_BLOBKEY = "profile-photo-blobkey";
```

Comment:
nit: can you add a comment above all these strings to clarify why they are duplicated in the test? vs. referencing the constants in production code

Reply:
How would I use the constants from production code. That seems like it would be better. I tried importing the constants.

Reply:
Yep, exactly.